applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -506,22 +507,22 @@ static void stream_started_cb(struct bt_audio_stream *stream)
 	ERR_CHK(ret);
 }
 
-static void stream_metadata_updated_cb(struct bt_audio_stream *stream)
+static void stream_metadata_updated_cb(struct bt_bap_stream *stream)
 {
 	LOG_DBG("Audio Stream %p metadata updated", (void *)stream);
 }
 
-static void stream_disabled_cb(struct bt_audio_stream *stream)
+static void stream_disabled_cb(struct bt_bap_stream *stream)
 {
 	LOG_DBG("Audio Stream %p disabled", (void *)stream);
 }
 
-static void stream_stopped_cb(struct bt_audio_stream *stream)
+static void stream_stopped_cb(struct bt_bap_stream *stream, uint8_t reason)
 {
 	int ret;
 	uint8_t channel_index;
 
-	LOG_INF("Stream %p stopped", (void *)stream);
+	LOG_INF("Stream %p stopped. Reason %d", (void *)stream, reason);
```

Comment:
Reason should be DBG, and perhaps the pointer to the stream. It could be better to print the channel_index actually, but willing to discuss

Reply:
As discussed on Teams, this will be addressed when we do the architecture rewrite